api/src/opentrons/protocol_engine/state/labware.py

```diff
@@ -48,6 +60,14 @@
 _INSTRUMENT_ATTACH_SLOT = DeckSlotName.SLOT_2
 
 
+class LabwareLoadParams(NamedTuple):
```

Comment:
niptick can we change this to:
```suggestion
class LoadLabwareParams(NamedTuple):
```

Reply:
As long as there's no danger of confusion with the `LoadLabwareParams` Pydantic model. No strong preference from me, either way.

Reply:
same. no strong preference!